GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/SubscriptionList.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public static class SubscriptionList
+{
+    public class Subscription
+    {
+        public Subscription(string name, SubscriptionType type, float price, GameObject subscribeButton)
+        {
+            Name = name;
+            Type = type;
+            Price = price;
+            SubscribeButton = subscribeButton;
+        }
+
+        public string Name { get; }
+
+        public SubscriptionType Type { get; }
+
+        public float Price { get; }
+
+        public GameObject SubscribeButton { get; }
+    }
+
+    public static readonly Subscription SilverSubscription =
+        new Subscription("silver VIP", SubscriptionType.SilverSubscription, 1.99f,
+            GameObject.Find("silverSubscriptionButton").gameObject);
+
+    public static readonly Subscription GoldenSubscription =
+        new Subscription("golden VIP", SubscriptionType.GoldenSubscription, 4.99f,
+            GameObject.Find("goldenSubscriptionButton").gameObject);
+
+    public static readonly Subscription NoSubscription =
+        new Subscription("No VIP", SubscriptionType.NoSubscription, 0f, new GameObject());
+
+    public static readonly List<Subscription> List = new List<Subscription>()
+        {NoSubscription, SilverSubscription, GoldenSubscription};
+
+    public static Subscription GetSubscriptionObjByType(SubscriptionType subscriptionType)
```

Comment:
If it is 1-to-1 mapping, I don't see much values of using an enum here. Why don't use the object directly?

Reply:
the reason is when I store the game data in local json file, it will be more convenient to store the enum, comparing to store the object.